apps/explorer/app/contract/[id]/page.tsx

```diff
@@ -76,13 +77,45 @@ export default async function Page({ params }) {
   const renewalTransaction = rt?.transaction
   const renewedTo = renewalTransaction?.storage_contracts?.[0]
 
+  // The following is a temporary addition to satisfy new Transaction
```

Comment:
Whats the plan with this temporary addition, swap everything out in a follow up PR?

Reply:
Correct, yes. Like all the routes so far, there is a fair amount of reconciliation that needs to occur between what Explorer asks for from Sia Central for a contract and what it presently can receive from explored for a contract.

This much code satisfies the Transaction component within ContractView. Another option is to temporarily remove that Transaction component from ContractView,